addons/mail/static/src/discuss/call/public_web/discuss_sidebar_call_participants.xml

```diff
@@ -62,34 +41,33 @@
     </t>
 
     <t t-name="mail.DiscussSidebarCallParticipants.participant">
-        <div class="o-mail-DiscussSidebarCallParticipants-participant d-flex text-reset overflow-hidden align-items-center" t-att-class="{ 'justify-content-center bg-inherit': isCompact }">
-            <div class="bg-inherit position-relative d-flex flex-shrink-0" style="width:24px;height:24px;margin:1px">
+        <div class="o-mail-DiscussSidebarCallParticipants-participant d-flex text-reset overflow-hidden align-items-center">
+            <div class="bg-inherit position-relative d-flex flex-shrink-0" style="width:24px;height:24px;padding:1px">
```

Comment:
`margin: 1px` must be kept (this is an intentional change, kinda unrelated to compact mode)